src/main/java/org/apache/bcel/verifier/structurals/Subroutines.java

```diff
@@ -121,16 +121,16 @@ public String toString() {
             ret.append("', Instructions: '").append(instructions).append("'.");
 
             ret.append(" Accessed local variable slots: '");
-            int[] alv = getAccessedLocalsIndices();
-            for (final int element : alv) {
-                ret.append(element);ret.append(" ");
+            int[] lvs = getAccessedLocalsIndices();
+            for (final int lv : lvs) {
```

Comment:
I do not see how trading an acronym for a mysterious TLA helps.

Reply:
To make things worse :-) this also trades a word "element" for a meaningless acronym "lv" in this context.

Reply:
The rationale is that "lvs" (apparently LocalVariables) is consistently used in several modules in verifier/structurals. "alv" was an outlier here.